spectre-meltdown-checker.sh

```diff
@@ -288,7 +288,9 @@ cve2name()
 		CVE-2019-11091) echo "RIDL, microarchitectural data sampling uncacheable memory (MDSUM)";;
 		CVE-2019-11135) echo "ZombieLoad V2, TSX Asynchronous Abort (TAA)";;
 		CVE-2018-12207) echo "No eXcuses, iTLB Multihit, machine check exception on page size changes (MCEPSC)";;
-		CVE-2020-0543) echo "Special Register Buffer Data Sampling (SRBDS)";;
+		CVE-2020-0543)  echo "Special Register Buffer Data Sampling (SRBDS)";;
```

Comment:
Also, here looks like you are mixing formatting fix with CVE-2022-0001 and CVE-2022-0002 checking commit

Reply:
Uggh. You are right. Thanks.